mockgcp/mockstorage/objects.go

```diff
@@ -44,7 +44,7 @@ func (s *objects) ListObjects(ctx context.Context, req *pb.ListObjectsRequest) (
 func (s *objects) GetObject(ctx context.Context, req *pb.GetObjectRequest) (*pb.Object, error) {
 	// A stub implementation, just to support deletion (for now)
 
-	httpmux.SetExpiresHeader(ctx, time.Now())
+	// httpmux.SetExpiresHeader(ctx, time.Now())
```

Comment:
Is this intended? Do you mind adding  a comment to explain what problem it gives, since the other objects still use setExpiresHeader

Reply:
Removed!

Reply:
Ah, I see.  The real GCP logs don't seem to override this header, so we don't want to set it in the mocks.